index.js

```diff
@@ -10,7 +10,7 @@ export * as commentsAsync from './adhocracy4/comments_async/static/comments_asyn
 export { default as config } from './adhocracy4/static/config'
 export { default as follows } from './adhocracy4/follows/static/follows/react_follows'
 export * as maps from './adhocracy4/maps/static/a4maps/a4maps_common'
-export { default as polls } from './adhocracy4/polls/assets/react_polls'
```

Comment:
@goapunk sorry missed this before, need to also rename the folder :p then feel free to merge

Reply:
sorry, don't understand?

Reply:
ahh, forgot to commit :man_facepalming: